src/unix/mod.rs

```diff
@@ -68,7 +68,12 @@ s! {
 
     pub struct timeval {
         pub tv_sec: time_t,
+        #[cfg(not(linux_time_bits64))]
         pub tv_usec: suseconds_t,
+        // For 64 bit time on 32 bit linux glibc, suseconds_t is still
+        // a 32 bit type.  Using suseconds_t here will break the tests, use i64 instead
+        #[cfg(linux_time_bits64)]
+        pub tv_usec: i64,
```

Comment:
How does this work in the kernel - is `suseconds_t` getting defined somewhere else? I only see https://github.com/torvalds/linux/blob/9f16d5e6f220661f73b36a4be1b21575651d8833/include/uapi/asm-generic/posix_types.h#L41 and something for sparc64.

Reply:
Ah, I see this is redefined in glibc https://github.com/bminor/glibc/blob/d92a5e1dad39ab0ac09ab1beeb055c1f1d981cd4/time/bits/types/struct_timeval.h#L8-L17. If the tests still pass, could you remove the changes to this type and leave a `// FIXME(time): ...` to adjust when we do the glibc updates?

Reply:
Done